docker-compose/study/docker-compose.override.yml

```diff
@@ -297,3 +297,21 @@ services:
     command: --server.port=80 --spring.config.additional-location=/config/
     sysctls:
       - net.ipv4.ip_unprivileged_port_start=0 # for docker < 20.03.0
+
+  voltage-init-server:
+    image: gridsuite/voltage-init-server:latest
+    ports:
+      - 5038:80
+    volumes:
+      - $PWD/../../k8s/resources/study/config/voltage-init-server-application.yml:/config/specific/application.yml:Z
```

Comment:
this new file voltage-init-server-application.yml should be created within this PR ?

Reply:
Yes done